api/hasura/src/utils/configUtils.ts

```diff
@@ -0,0 +1,43 @@
+import assert from 'assert';
```

Comment:
Use [strict mode](https://nodejs.org/docs/latest-v10.x/api/assert.html#assert_strict_mode) instead of legacy 

```suggestion
import { strict as assert } from 'assert';
```